tests/testdata/config/none.yaml

```diff
@@ -7,8 +11,155 @@
 
 # TODO: the IP addresses are not namespaced yet, so must be unique on the mesh (flat namespace) including in
 # ServiceEntry tests. Removing deps on ip in progress.
+---
+# Default sidecar
+apiVersion: networking.istio.io/v1alpha3
+kind: Sidecar
+metadata:
+  name: default
+  namespace: none
+spec:
+  egress:
+    - hosts:
+        # TODO: if not imported, disable mixer checks
+        - istio-system/istio-policy.istio-system
+        - istio-system/istio-mixer.istio-system # Doesn't work
+        - istio-system/* # WORKS
+        - none/*
+        - default/test.default # TODO: without namespace it  fails validation !
+      # TODO: if we include the namespace, why do we need full name ? Importing regular services should work.
+
+  # Label selection seems to confuse the new code.
+  ingress:
+    - port:
+        number: 7071
+        protocol: HTTP
+        name: httplocal
+      default_endpoint: 127.0.0.1:17071
```

Comment:
Should this be `defaultEndpoint` - https://preliminary.istio.io/docs/reference/config/istio.networking.v1alpha3/#IstioIngressListener

Reply:
both work AFAIK. Probably defaultEndpoint is the correct one.